packages/swap/src/utils/dataStructures.ts

```diff
@@ -17,7 +17,7 @@ class Timer<T> {
   }
 
   getValue(): T {
-    return this.reset().value;
```

Comment:
i think the `reset` is necessary to refresh the timeout (and keep the cache value alive)

Reply:
The issue with having a reset here.. means that the cache will continue to hit if we have a ton of traffic.  Its not right for every case. In the processor we already cache by block specific attributes with high timeouts so its not really an issue.

Reply:
i see, makes sense in this case. hope we dont break any other place where we use the `CacheMap`